tests/sklearn/test_sklearn_models.py

```diff
@@ -1658,7 +1729,11 @@ def test_p_error_simulation(
     The test checks that models compiled with a large p_error value predicts very different results
     with simulation or in FHE compared to the expected clear quantized ones.
     """
-    n_bits = get_n_bits_non_correctness(model_class)
+
+    if os.getenv("TREES_USE_FHE_SUM") == "1":
+        n_bits = 4
```

Comment:
if only works for 4 bits ? also maybe better to include this in directly in `get_n_bits_non_correctness` and be sure to check that this is only applied for tree-based models (using `model_class`) !

Reply:
I reduced the precision because with the sum in fhe the bitwidth is larger and I have an OOM on my machine.

Reply:
done